docs/distribution/packaging.md

```diff
@@ -159,6 +159,17 @@ version: <semver string of the required library version>
 >
 > - Extend the library version field to handle version bounds.
 
+### The `visualization` Directory
+
+As Enso is a visual language, the package may contain specification how 
+data can be displayed in various tools, e.g. 
+[Enso IDE](https://github.com/enso-org/ide). Conventionally, all those files
+should be put in the `visualization` directory, and from there all tools should
+read them.
```

Comment:
```suggestion
As Enso is a visual language, a package may contain a specification of how 
data can be displayed in various tools, for example [Enso IDE](https://github.com/enso-org/ide). 
The Enso package structure may optionally contain a `visualization` directory
which may contain visualization definitions.
```

You need to add this folder into the directory tree described above, and make sure that it is marked as optional.